server/config/config.go

```diff
@@ -54,13 +54,13 @@ const (
 	// GrpcServiceKeepAlivePingMinIntervalSec controls the min interval for one keepalive ping.
 	defaultGrpcServiceKeepAlivePingMinIntervalSec int = 20
 
-	defaultNodeNamePrefix          = "ceresmeta"
+	defaultNodeNamePrefix          = "horaemeta"
```

Comment:
There may be a compatibility issue here.

Reply:
During the meta upgrade process, the new and old nodes are unable to discover each other, but it does not affect the overall functionality.